src/app/content/highlights/components/ShowMyHighlightsStyles.tsx

```diff
@@ -23,6 +24,12 @@ export const HighlightsChapter = styled.div`
 // tslint:disable-next-line:variable-name
 export const ShowMyHighlightsBody = styled(PopupBody)`
   padding: 0;
+  background: ${theme.color.neutral.darker};
+
+  ${Filters} {
+    padding: 0 3.2rem;
```

Comment:
i think it would simplify life if `PopupBody` didn't have padding, so that filters could have the same styles in both places